src/Erdene/Finra.java

```diff
@@ -0,0 +1,20 @@
+package Erdene;
+
+public class Finra {
+    public static void finra(int number){
```

Comment:
Task was:
Write a function which prints out the numbers from 1 to 30 but for numbers which are a multiple of 3,
print "FIN" instead of the number and for numbers which are a multiple of 5, print "RA" instead of the number.
For numbers which are a multiple of both 3 and 5, print "FINRA" instead of the number.

So that method do something different

Task was:
Write a function which prints out the numbers from 1 to 30 but for numbers which are a multiple of 3,
print "FIN" instead of the number and for numbers which are a multiple of 5, print "RA" instead of the number.
For numbers which are a multiple of both 3 and 5, print "FINRA" instead of the number.

So that method do something different